skops/card/_model_card.py

```diff
@@ -1410,13 +1425,18 @@ def save(self, path: str | Path) -> None:
         path: str, or Path
             Filepath to save your card.
 
+        plot_path: str
+            Filepath to save the plots.
+
         Notes
         -----
         The keys in model card metadata can be seen `here
         <https://huggingface.co/docs/hub/models-cards#model-card-metadata>`__.
         """
         with open(path, "w", encoding="utf-8") as f:
-            f.write("\n".join(self._generate_card()))
+            if not isinstance(path, Path):
+                path = Path(path)
+            f.write("\n".join(self._generate_card(path.parent if copy_files else None)))
```

Comment:
Just a nit, but this is more readable IMO:

```suggestion
            destination_path = path.parent if copy_files else None
            f.write("\n".join(self._generate_card(destination_path=destination_path)))
```